server/plugins/index.js

```diff
@@ -159,6 +161,8 @@ checkStatus()
 			'Initializing Plugin Server...'
 		);
 
+		unstakingCheckRunner();
```

Comment:
@fetok12 can you put these into the job file and avoid mixing these with the plugins. So the functions basically run in the job file. Once job is imported they would run there so no need to have these here.